x-pack/solutions/observability/plugins/synthetics/server/synthetics_service/private_location/synthetics_private_location.ts

```diff
@@ -65,7 +65,7 @@ export class SyntheticsPrivateLocation {
     return newPolicy;
   }
 
-  getPolicyId(config: HeartbeatConfig, locId: string, spaceId: string) {
+  getPolicyId(config: { origin?: string; id: string }, locId: string, spaceId: string) {
```

Comment:
Why is this marked as optional? I'd be concerned we can't accurately know the the id format this way.

Reply:
origin type is marked optional in monitor fields, so i can't escape that.